support/jwt/src/main/kotlin/org/doorip/support/jwt/RSAPublicKey.kt

```
@@ -0,0 +1,34 @@
+            val nBytes = decoder.decode(n)
+            val eBytes = decoder.decode(e)
+
+            val n = BigInteger(1, nBytes)
+            val e = BigInteger(1, eBytes)
+
+            val rsaPublicKeySpec = RSAPublicKeySpec(n, e)
+
+            val keyFactory = KeyFactory.getInstance(type)
+
+            keyFactory.generatePublic(rsaPublicKeySpec)
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)